### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/db/nedb.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ class DBHelper {
     let fieldsObj;
```
Comment (alex-cory) [inline src/db/nedb.js]: why don't we use reduce here?
```js
const fieldsObj = !_.isEmpty(fields) && fields.reduce((field, obj) => obj[field] = 1, {});
```

File: src/route/api.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const CentralizedOracle = require('../api/centralized_oracle');
 const DecentralizedOracle = require('../api/decentralized_oracle');
 const Transaction = require('../api/transaction');
 const QtumUtils = require('../api/qtum_utils');
-const Emitter = require('../utils/emitterHelper');
+const { showSaveDialog, showImportDialog } = require('../utils/emitterHelper');
```
Comment (alex-cory) [inline src/route/api.js]: way sexier 🤤

File: src/db/nedb.js (hunk 1 above)
Comment (dwalintukan) [reply src/db/nedb.js]: can change this in a different PR. my purpose for this one is just to fix the tests. ran the linter so this change in this file showed up.

**Comment by dwalintukan** [comment]
Closing this PR. Tests are already fixed in `bodhi-server`

**Pull request closed without merging**